api/types/accesslist/accesslist.go

```diff
@@ -41,6 +41,9 @@ const (
 	OneYear     ReviewFrequency = 12
 
 	twoWeeks = 24 * time.Hour * 14
+
+	// NoMemberCount is a special value that indicates no members count is included in the access list.
+	NoMemberCount = -1
```

Comment:
Either the "no value" value must be zero, or the field must be marked `optional` and it's going to rely on explicit presence; otherwise, servers sending an `AccessListSpec` message that don't know about the field will implicitly send a valueful zero.